### PR TITLE
feat(Stories): add hook for sync tabs

### PR DESCRIPTION
**Pull request by eyoue**
UXRFC-300.
Perform action in multiple tabs simultaneously using Local Storage key and 'storage' event listener.

### CODE REVIEW

File: src/components/Stories/Stories.tsx [hunk 1]
```diff
@@ -48,13 +51,21 @@ export function Stories({
         [onClose],
     );
 
+    const {callback: closeWithLS} = useSyncWithLS<NonNullable<StoriesProps['onClose']>>({
+        callback: (event, reason) => {
+            if (syncInTabs) onClose?.(event, reason);
```
Comment (DarkGenius) [inline src/components/Stories/Stories.tsx:56]: Is it necessary to check `syncInTabs` here?

File: src/components/Stories/__stories__/Stories.stories.tsx [hunk 2]
```diff
@@ -62,6 +62,7 @@ const DefaultTemplate: StoryFn<StoriesProps> = (props: StoriesProps) => {
             <Stories
                 {...props}
                 open={visible}
+                syncInTabs={props.syncInTabs}
```
Comment (DarkGenius) [inline src/components/Stories/__stories__/Stories.stories.tsx:65]: Above, with the restructuring of props, `syncInTabs` also will be set, right?

File: src/components/Stories/Stories.tsx (hunk 1 above)
Comment (eyoue) [reply src/components/Stories/Stories.tsx]: Tnx, fixed.

File: src/components/Stories/__stories__/Stories.stories.tsx (hunk 2 above)
Comment (eyoue) [reply src/components/Stories/__stories__/Stories.stories.tsx]: yes, right. Fixed.

**Review by DarkGenius — APPROVED** [review]

**Review by burashka — APPROVED** [review]

**Review by vanilla-wave — APPROVED** [review]

**Pull request merged**